src/com/google/enterprise/adaptor/database/DatabaseAdaptor.java

```diff
@@ -273,12 +273,16 @@ public void init(AdaptorContext context) throws Exception {
 
     // Verify all column names.
     try (Connection conn = makeNewConnection()) {
-      verifyColumnNames(conn, "db.everyDocIdSql", everyDocIdSql,
-          "db.uniqueKey", uniqueKey.getDocIdSqlColumns());
-      verifyColumnNames(conn, "db.singleDocContentSql", singleDocContentSql,
-          "db.singleDocContentSqlParameters", uniqueKey.getContentSqlColumns());
-      verifyColumnNames(conn, "db.aclSql", aclSql,
-          "db.aclSqlParameters", uniqueKey.getAclSqlColumns());
+      uniqueKey.addColumnTypes(
```

Comment:
I'm usually a fan of nesting calls, but not when the *primary effect* of the buried call is a side-effect, as in this case. I pondered a local Map variable, but I think I prefer passing uniqueKey (or null) to verifyColumnNames, and having verifyColumnNames push the map.